src/pages/organize/[orgId]/projects/[campId]/emails/[emailId]/design/index.tsx

```diff
@@ -24,15 +25,32 @@ export const getServerSideProps: GetServerSideProps = scaffold(
   }
 );
 
-const EmailPage: PageWithLayout = () => {
+type Props = {
+  campId: string;
```

Comment:
I don't think I am! I'm just adding it to the component props because it's a page component. But I'm guessing the reason you're asking is because it's unnecessary to have it in the type when it's not used, and I guess you're right!